tox.ini

```diff
@@ -26,98 +26,106 @@ commands =
 [testenv:py3.7]
 basepython = python3.7
 deps =
-    pipenv
-    pytest
-    pytest-cov
-    pytest-asyncio
-    gym
-    numpy
-    tensorflow
-
+    pytest==5.3.5
+    pytest-cov==2.8.1
+    pytest-asyncio==0.10.0
+    docker
+    colorlog==4.1.0
+    oef==0.8.1
+    gym==0.15.6
+    numpy==1.18.1
+    tensorflow==1.14.0
 commands =
-    pipenv install
-    pip install -e .[all]
```

Comment:
isn't `pip install -e .[all]` required?

Reply:
it is not required. `tox` provides to install the package by default.
However, a problem I'm encountering is that the coverage is not computed only on the main code base but also on the copy of the codebase `tox` does.